plaso/data/formatters/android.yaml

```diff
@@ -12,6 +12,17 @@ short_source: 'LOG'
 source: 'Android App Usage'
 ---
 type: 'conditional'
+data_type: 'android:event:battery'
+message:
```

Comment:
given that both battery_saver and charge_type are enumerations, let's add their values here